backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/impexp/BankStatementImportTableSqlUpdater.java

```diff
@@ -169,7 +169,10 @@ private void updateOrgBankAccount(final BankAccountId orgBankAccountId, final Im
 																+ " WHERE a." + I_C_BP_BankAccount.COLUMNNAME_IBAN
 																+ " = i." + I_I_BankStatement.COLUMNNAME_IBAN
 																+ " AND a.AD_Client_ID=i.AD_Client_ID "
-																+ " )"
+																+ " AND i." + I_I_BankStatement.COLUMNNAME_AD_Org_ID
```

Comment:
please take into account that a bpartner and thus its bankaccount might also have AD_Org_ID=0